tests/lib/init/config-initializer.js

```diff
@@ -207,6 +207,84 @@ describe("configInitializer", () => {
             });
         });
 
+        describe("writeFile()", () => {
```

Comment:
I think this shouldn't be nested in `describe("processAnswers())")` because it's a separate function.